tests/QARegressionTests/test_Issues/test_CoreIssue576.py

```diff
@@ -22,48 +31,36 @@ def test_negative_dataset():
     stdout, stderr = process.communicate()
 
     file_name = stdout.decode().strip().split(": ")[1] + ".xlsx"
-    print(file_name)
     # Open the Excel file
     workbook = openpyxl.load_workbook(file_name)
 
     # Go to the "Issue Details" sheet
-    sheet = workbook["Issue Details"]
+    Issues_Sheet = workbook["Issue Details"]
+    Dataset_Sheet = workbook["Dataset Details"]
+    Rules_Sheet = workbook["Rules Report"]
 
-    record_column = sheet["F"]
-    variables_column = sheet["H"]
-    values_column = sheet["I"]
+    COREID_Column = Issues_Sheet["A"]
```

Comment:
We typically follow PEP 8 style guide, which says variable names should be lowercase, with words separated by underscores

https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
updated